app/code/Magento/ProductAlert/Controller/Unsubscribe/Stock.php

```diff
@@ -79,4 +100,4 @@ public function execute()
         $resultRedirect->setUrl($product->getProductUrl());
         return $resultRedirect;
     }
-}
+}
```

Comment:
Please return empty new line in to the end of the file according to [PSR-2 2.2](https://www.php-fig.org/psr/psr-2/#22-files)

Reply:
added